config-model-api/src/main/java/com/yahoo/config/model/api/ModelContext.java

```diff
@@ -54,8 +54,8 @@
         boolean isBootstrap();
         boolean isFirstTimeDeployment();
         boolean useDedicatedNodeForLogserver();
-        boolean useFdispatchByDefault();
-        boolean dispatchWithProtobuf();
+        boolean useFdispatchByDefault(); // TODO Remove once 7.110 is the oldest old config models
```

Comment:
Add default implementation returning `false` and `true` respectively.

Reply:
That is not an API change ?

Reply:
Not a public API

Reply:
Confirmed, with @hmusum that is was fine.

Reply:
Done